go/ct/gen/interval_solver_test.go

```diff
@@ -285,15 +285,12 @@ func TestIntervalSolver_uint64fullrangeAndEdges(t *testing.T) {
 		setup func(*IntervalSolver[uint64])
 		check func(uint64) bool
 	}{
-		"full-range": {
```

Comment:
I think there is still value in checking that the default is the whole range. Do you think we can test this in a different manner instead of just deleting the test ?

Reply:
You cannot check if it is not in rage, since the test returns uint64 and the value is uint64, any out of range would overflow and therefore be in range. 
Nevertheless, testing that this code does not return an err